src/raps/actions/unlock.js

```diff
@@ -54,17 +54,15 @@ const unlock = async (wallet, currentRap, index, parameters) => {
 
   console.log('[unlock] add a new txn');
   dispatch(
-    dataAddNewTransaction(
-      {
-        amount: 0,
-        asset: assetToUnlock,
-        from: wallet.address,
-        hash: approval.hash,
-        nonce: get(approval, 'nonce'),
-        to: get(approval, 'to'),
-      },
-      true
-    )
+    dataAddNewTransaction({
+      amount: 0,
+      asset: assetToUnlock,
+      from: wallet.address,
+      hash: approval.hash,
+      nonce: get(approval, 'nonce'),
+      to: get(approval, 'to'),
+      type: 'authorize',
```

Comment:
Can we use the transactionType instead of the string 'authorize' - also, can you include the status: transactionStatusType (authorizing)